meilisearch-http/public/interface.html

```diff
@@ -221,7 +221,7 @@ <h2 class="subtitle">
                 results.innerHTML = '';
 
                 let processingTimeMs = httpResults.processingTimeMs;
-                let numberOfDocuments = httpResults.hits.length;
+                let numberOfDocuments = httpResults.nbHits;
```

Comment:
this is the total number of documents that matched the query, is that what you want?

Reply:
yes, I thought it makes more sense to have the total number of matched documents instead of the number of documents returned by the engine which is very often 20 because of the default limit. WDYT?

I would also like the input of @qdequele here :)

Reply:
Yes, I think also that is better.